File: submissions/sm_108_krishna-kant/week_03/day_3/session_2/split_string.js
var givenString = "Hey My Name Is Krishna Kant Sharma";
var splitCharacter = "k"; // I am taking space you can choose either any letter
var stringArray = [];
function splitString(args) {
  var partialString = "";
  for (i = 0; i < args.length; i++) {
    if (args[i] === splitCharacter) {
      stringArray.push(partialString);
      partialString = "";
    } else {
      partialString += args[i];
    }
  }
  stringArray.push(partialString);
  console.log(stringArray);
}

splitString(givenString.toLowerCase());

File: submissions/sm_108_krishna-kant/week_13/day_4/coding_1/split_string.py
givenString = "Hello My Name Is Krishna"
key = " "
splittedString = [] 
tempString = ""
for i in givenString:
    if i == key:
        splittedString.append(tempString)
        tempString = ""
    else:
        tempString += i
splittedString.append(tempString)
print(splittedString)